packages/react/src/components/NumberInput/NumberInput.tsx

```diff
@@ -1040,6 +1040,8 @@ const Label: FC<Label> = ({ disabled, id, hideLabel, label }) => {
   return null;
 };
 
+// TODO: `Label` isn't exported. Why does it have `propTypes`? I have the same
+// question for the `HelperText` component too.
```

Comment:
Best practice for runtime checks before we had ts? idk honestly, probably to fix an old react or eslint warning or something about missing proptypes. You can remove if you'd like or leave in.